airbyte-ci/connectors/metadata_service/lib/metadata_service/models/src/ConnectorRegistryDestinationDefinition.yaml

```diff
@@ -47,6 +47,11 @@ properties:
     description: The date when this connector was first released, in yyyy-mm-dd format.
     type: string
     format: date
+  tags:
+    type: array
+    description: "An array of tags that describe the connector. E.g: language:python, keyword:rds, etc."
+    items:
+      type: string
```

Comment:
do you want to enforce the "key:value" thing?  Perhaps a regular expression can be used for that.

Reply:
updated!